### ISSUE TITLE
unnecessary downgrade of numpy with conda 4.6.2

### ISSUE THREAD

**Issue by wdirons** [conda/conda#8236]
## Current Behavior
`conda install 'numpy>=1.12.0'` with numpy already installed when using conda 4.6.2 will downgrade numpy. 

### Steps to Reproduce

Installed https://repo.continuum.io/archive/Anaconda3-2018.12-Linux-ppc64le.sh on an ubuntu:18.04 container
```
conda create -n test_env python=3.6 numpy
conda activate test_env
conda install 'numpy>=1.12.0'
conda update -n base -c defaults conda
conda install 'numpy>=1.12.0'
```

**When 4.5.12 was installed, numpy was ok:**
```
(test_env) root@fd9798e8557f:~# conda install 'numpy>=1.12.0'
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.5.12
  latest version: 4.6.2

Please update conda by running

    $ conda update -n base -c defaults conda



# All requested packages already installed.
```
**Upgrade to 4.6.2**
```
(test_env) root@fd9798e8557f:~# conda update -n base -c defaults conda
Solving environment: done

## Package Plan ##

  environment location: /root/anaconda3

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.6.2                |           py37_0         1.7 MB

The following packages will be UPDATED:

    conda: 4.5.12-py37_0 --> 4.6.2-py37_0

Proceed ([y]/n)? y


Downloading and Extracting Packages
conda-4.6.2          | 1.7 MB    | ########################################################################################################################################################################## | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```
**Retry the same command to install numpy>=1.12.0, and numpy will be downgraded.**
```
(test_env) root@fd9798e8557f:~# conda install 'numpy>=1.12.0'
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /root/anaconda3/envs/test_env

  added / updated specs:
    - numpy[version='>=1.12.0']


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    numpy-1.12.0               |           py36_0        10.3 MB
    ------------------------------------------------------------
                                           Total:        10.3 MB

The following packages will be REMOVED:

  blas-1.0-openblas
  numpy-base-1.15.4-py36h2f8d375_0

The following packages will be SUPERSEDED by a higher-priority channel:

  numpy              pkgs/main::numpy-1.15.4-py36h99e49ec_0 --> pkgs/free::numpy-1.12.0-py36_0


Proceed ([y]/n)? n


Exiting.

(test_env) root@fd9798e8557f:~#

```

## Expected Behavior
The behavior with conda 4.5.12 was it did nothing because numpy was already installed. 

## Environment Information

Commands run **before** the upgrade to 4.6.2
<details open><summary><code>`conda info`</code></summary><p>

```
root@fd9798e8557f:~# conda info

     active environment : base
    active env location : /root/anaconda3
            shell level : 1
       user config file : /root/.condarc
 populated config files :
          conda version : 4.5.12
    conda-build version : 3.16.3
         python version : 3.7.1.final.0
       base environment : /root/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-ppc64le
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-ppc64le
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-ppc64le
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-ppc64le
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /root/anaconda3/pkgs
                          /root/.conda/pkgs
       envs directories : /root/anaconda3/envs
                          /root/.conda/envs
               platform : linux-ppc64le
             user-agent : conda/4.5.12 requests/2.21.0 CPython/3.7.1 Linux/4.14.0-115.2.2.el7a.ppc64le ubuntu/18.04 glibc/2.27
                UID:GID : 0:0
             netrc file : None
           offline mode : False

```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
root@fd9798e8557f:~# conda config --show-sources

root@fd9798e8557f:~#

```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
root@fd9798e8557f:~# conda list --show-channel-urls
# packages in environment at /root/anaconda3:
#
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0                    py37_0    defaults
alabaster                 0.7.12                   py37_0    defaults
anaconda                  2018.12                  py37_0    defaults
anaconda-client           1.7.2                    py37_0    defaults
anaconda-project          0.8.2                    py37_0    defaults
asn1crypto                0.24.0                   py37_0    defaults
astroid                   2.1.0                    py37_0    defaults
astropy                   3.0.5            py37h7b6447c_0    defaults
atomicwrites              1.2.1                    py37_0    defaults
attrs                     18.2.0           py37h28b3542_0    defaults
babel                     2.6.0                    py37_0    defaults
backcall                  0.1.0                    py37_0    defaults
backports                 1.0                      py37_1    defaults
backports.os              0.1.1                    py37_0    defaults
backports.shutil_get_terminal_size 1.0.0                    py37_2    defaults
beautifulsoup4            4.6.3                    py37_0    defaults
bitarray                  0.8.3            py37h14c3975_0    defaults
bkcharts                  0.2                      py37_0    defaults
blas                      1.0                    openblas    defaults
bleach                    3.0.2                    py37_0    defaults
blosc                     1.14.4               hdbcaa40_0    defaults
bokeh                     1.0.2                    py37_0    defaults
boto                      2.49.0                   py37_0    defaults
bottleneck                1.2.1            py37h035aef0_1    defaults
bzip2                     1.0.6                h14c3975_5    defaults
ca-certificates           2018.03.07           hf82bc7d_0    defaults
cairo                     1.14.12              h8948797_3    defaults
certifi                   2018.11.29               py37_0    defaults
cffi                      1.11.5           py37h9745a5d_0    defaults
chardet                   3.0.4                    py37_1    defaults
click                     7.0                      py37_0    defaults
cloudpickle               0.6.1                    py37_0    defaults
clyent                    1.2.2                    py37_1    defaults
colorama                  0.4.1                    py37_0    defaults
conda                     4.5.12                   py37_0    defaults
conda-build               3.16.3                   py37_0    defaults
conda-env                 2.6.0                         1    defaults
conda-verify              3.1.1                    py37_0    defaults
contextlib2               0.5.5                    py37_0    defaults
cryptography              2.4.2            py37h1ba5d50_0    defaults
curl                      7.63.0            hbc83047_1000    defaults
cycler                    0.10.0                   py37_0    defaults
cython                    0.29.2           py37he6710b0_0    defaults
cytoolz                   0.9.0.1          py37h14c3975_1    defaults
dask                      1.0.0                    py37_0    defaults
dask-core                 1.0.0                    py37_0    defaults
datashape                 0.5.4                    py37_1    defaults
decorator                 4.3.0                    py37_0    defaults
distributed               1.25.1                   py37_0    defaults
docutils                  0.14                     py37_0    defaults
entrypoints               0.2.3                    py37_2    defaults
et_xmlfile                1.0.1                    py37_0    defaults
expat                     2.2.6                he6710b0_0    defaults
fastcache                 1.0.2            py37h14c3975_2    defaults
filelock                  3.0.10                   py37_0    defaults
flask                     1.0.2                    py37_1    defaults
flask-cors                3.0.7                    py37_0    defaults
fontconfig                2.13.0               h9420a91_0    defaults
freetype                  2.9.1                h8a8886c_0    defaults
future                    0.17.1                   py37_0    defaults
get_terminal_size         1.0.0                         0    defaults
gevent                    1.3.7            py37h7b6447c_1    defaults
glib                      2.56.2               hd408876_0    defaults
glob2                     0.6                      py37_1    defaults
gmp                       6.1.2                h7f7056e_2    defaults
gmpy2                     2.0.8            py37h10f8cd9_2    defaults
greenlet                  0.4.15           py37h7b6447c_0    defaults
h5py                      2.8.0            py37h8d01980_0    defaults
hdf5                      1.10.2               hba1933b_1    defaults
heapdict                  1.0.0                    py37_2    defaults
html5lib                  1.0.1                    py37_0    defaults
icu                       58.2                 h64fc554_1    defaults
idna                      2.8                      py37_0    defaults
imageio                   2.4.1                    py37_0    defaults
imagesize                 1.1.0                    py37_0    defaults
importlib_metadata        0.6                      py37_0    defaults
ipykernel                 5.1.0            py37h39e3cac_0    defaults
ipython                   7.2.0            py37h39e3cac_0    defaults
ipython_genutils          0.2.0                    py37_0    defaults
ipywidgets                7.4.2                    py37_0    defaults
isort                     4.3.4                    py37_0    defaults
itsdangerous              1.1.0                    py37_0    defaults
jbig                      2.1                  h14c3975_0    defaults
jdcal                     1.4                      py37_0    defaults
jedi                      0.13.2                   py37_0    defaults
jinja2                    2.10                     py37_0    defaults
jpeg                      9b                   hcb7ba68_2    defaults
jsonschema                2.6.0                    py37_0    defaults
jupyter                   1.0.0                    py37_7    defaults
jupyter_client            5.2.4                    py37_0    defaults
jupyter_console           6.0.0                    py37_0    defaults
jupyter_core              4.4.0                    py37_0    defaults
jupyterlab                0.35.3                   py37_0    defaults
jupyterlab_server         0.2.0                    py37_0    defaults
kiwisolver                1.0.1            py37hf484d3e_0    defaults
krb5                      1.16.1               h597af5e_7    defaults
lazy-object-proxy         1.3.1            py37h14c3975_2    defaults
libarchive                3.3.3                h5d8350f_5    defaults
libcurl                   7.63.0            h20c2e04_1000    defaults
libedit                   3.1.20170329         h6b74fdf_2    defaults
libffi                    3.2.1                hf62a594_5    defaults
libgcc-ng                 8.2.0                h822a55f_1    defaults
libgfortran-ng            7.3.0                h822a55f_1    defaults
libopenblas               0.2.20               h9ac9557_7    defaults
libpng                    1.6.35               hbc83047_0    defaults
libsodium                 1.0.16               h1bed415_0    defaults
libssh2                   1.8.0                h1ba5d50_4    defaults
libstdcxx-ng              8.2.0                h822a55f_1    defaults
libtiff                   4.0.9                he85c1e1_1    defaults
libuuid                   1.0.3                h1bed415_2    defaults
libxcb                    1.13                 h1bed415_0    defaults
libxml2                   2.9.8                h26e45fe_1    defaults
libxslt                   1.1.32               h1312cb7_0    defaults
locket                    0.2.0                    py37_1    defaults
lxml                      4.2.5            py37hefd8a0e_0    defaults
lz4-c                     1.8.1.2              h14c3975_0    defaults
lzo                       2.10                 h0dabc4d_2    defaults
markupsafe                1.1.0            py37h7b6447c_0    defaults
matplotlib                3.0.2            py37h5429711_0    defaults
mccabe                    0.6.1                    py37_1    defaults
mistune                   0.8.4            py37h7b6447c_0    defaults
more-itertools            4.3.0                    py37_0    defaults
mpc                       1.1.0                h10f8cd9_1    defaults
mpfr                      4.0.1                hdf1c602_3    defaults
mpmath                    1.1.0                    py37_0    defaults
msgpack-python            0.5.6            py37h6bb024c_1    defaults
multipledispatch          0.6.0                    py37_0    defaults
nbconvert                 5.3.1                    py37_0    defaults
nbformat                  4.4.0                    py37_0    defaults
ncurses                   6.1                  he6710b0_1    defaults
networkx                  2.2                      py37_1    defaults
nltk                      3.4                      py37_1    defaults
nose                      1.3.7                    py37_2    defaults
notebook                  5.7.4                    py37_0    defaults
numexpr                   2.6.7            py37h98f89e9_0    defaults
numpy                     1.14.5           py37h2aefc1b_4    defaults
numpy-base                1.14.5           py37h2b20989_4    defaults
numpydoc                  0.8.0                    py37_0    defaults
odo                       0.5.1                    py37_0    defaults
olefile                   0.46                     py37_0    defaults
openblas                  0.2.20                        7    defaults
openblas-devel            0.2.20                        7    defaults
openpyxl                  2.5.12                   py37_0    defaults
openssl                   1.1.1a               h7b6447c_0    defaults
packaging                 18.0                     py37_0    defaults
pandas                    0.23.4           py37h04863e7_0    defaults
pandoc                    2.0.0.1                       1    defaults
pandocfilters             1.4.2                    py37_1    defaults
parso                     0.3.1                    py37_0    defaults
partd                     0.3.9                    py37_0    defaults
patchelf                  0.9                  he6710b0_3    defaults
path.py                   11.5.0                   py37_0    defaults
pathlib2                  2.3.3                    py37_0    defaults
patsy                     0.5.1                    py37_0    defaults
pcre                      8.42                 hf484d3e_0    defaults
pep8                      1.7.1                    py37_0    defaults
pexpect                   4.6.0                    py37_0    defaults
pickleshare               0.7.5                    py37_0    defaults
pillow                    5.3.0            py37h0d2faf8_0    defaults
pip                       18.1                     py37_0    defaults
pixman                    0.34.0               h1f8d8dc_3    defaults
pkginfo                   1.4.2                    py37_1    defaults
pluggy                    0.8.0                    py37_0    defaults
ply                       3.11                     py37_0    defaults
prometheus_client         0.5.0                    py37_0    defaults
prompt_toolkit            2.0.7                    py37_0    defaults
psutil                    5.4.8            py37h7b6447c_0    defaults
ptyprocess                0.6.0                    py37_0    defaults
py                        1.7.0                    py37_0    defaults
pycodestyle               2.4.0                    py37_0    defaults
pycosat                   0.6.3            py37h14c3975_0    defaults
pycparser                 2.19                     py37_0    defaults
pycrypto                  2.6.1            py37h14c3975_9    defaults
pycurl                    7.43.0.2         py37h1ba5d50_0    defaults
pyflakes                  2.0.0                    py37_0    defaults
pygments                  2.3.1                    py37_0    defaults
pylint                    2.2.2                    py37_0    defaults
pyodbc                    4.0.25           py37he6710b0_0    defaults
pyopenssl                 18.0.0                   py37_0    defaults
pyparsing                 2.3.0                    py37_0    defaults
pysocks                   1.6.8                    py37_0    defaults
pytables                  3.4.4            py37ha205bf6_0    defaults
pytest                    4.0.2                    py37_0    defaults
pytest-arraydiff          0.3              py37h39e3cac_0    defaults
pytest-astropy            0.5.0                    py37_0    defaults
pytest-doctestplus        0.2.0                    py37_0    defaults
pytest-openfiles          0.3.1                    py37_0    defaults
pytest-remotedata         0.3.1                    py37_0    defaults
python                    3.7.1                h4134adf_7    defaults
python-dateutil           2.7.5                    py37_0    defaults
python-libarchive-c       2.8                      py37_6    defaults
pytz                      2018.7                   py37_0    defaults
pywavelets                1.0.1            py37hdd07704_0    defaults
pyyaml                    3.13             py37h14c3975_0    defaults
pyzmq                     17.1.2           py37h14c3975_0    defaults
readline                  7.0                  h7b6447c_5    defaults
requests                  2.21.0                   py37_0    defaults
ruamel_yaml               0.15.46          py37h14c3975_0    defaults
scikit-image              0.14.1           py37he6710b0_0    defaults
scikit-learn              0.19.1           py37h6cfcb94_0    defaults
scipy                     1.1.0            py37h9c1e066_0    defaults
seaborn                   0.9.0                    py37_0    defaults
send2trash                1.5.0                    py37_0    defaults
setuptools                40.6.3                   py37_0    defaults
simplegeneric             0.8.1                    py37_2    defaults
singledispatch            3.4.0.3                  py37_0    defaults
six                       1.12.0                   py37_0    defaults
snappy                    1.1.7                h1532aa0_3    defaults
snowballstemmer           1.2.1                    py37_0    defaults
sortedcollections         1.0.1                    py37_0    defaults
sortedcontainers          2.1.0                    py37_0    defaults
sphinx                    1.8.2                    py37_0    defaults
sphinxcontrib             1.0                      py37_1    defaults
sphinxcontrib-websupport  1.1.0                    py37_1    defaults
sqlalchemy                1.2.15           py37h7b6447c_0    defaults
sqlite                    3.26.0               h7b6447c_0    defaults
statsmodels               0.9.0            py37h035aef0_0    defaults
sympy                     1.3                      py37_0    defaults
tblib                     1.3.2                    py37_0    defaults
terminado                 0.8.1                    py37_1    defaults
testpath                  0.4.2                    py37_0    defaults
tk                        8.6.8                hbc83047_0    defaults
toolz                     0.9.0                    py37_0    defaults
tornado                   5.1.1            py37h7b6447c_0    defaults
tqdm                      4.28.1           py37h28b3542_0    defaults
traitlets                 4.3.2                    py37_0    defaults
unicodecsv                0.14.1                   py37_0    defaults
unixodbc                  2.3.7                h2c717c6_0    defaults
urllib3                   1.24.1                   py37_0    defaults
wcwidth                   0.1.7                    py37_0    defaults
webencodings              0.5.1                    py37_1    defaults
werkzeug                  0.14.1                   py37_0    defaults
wheel                     0.32.3                   py37_0    defaults
widgetsnbextension        3.4.2                    py37_0    defaults
wrapt                     1.10.11          py37h14c3975_2    defaults
xlrd                      1.2.0                    py37_0    defaults
xlsxwriter                1.1.2                    py37_0    defaults
xlwt                      1.3.0                    py37_0    defaults
xz                        5.2.4                h14c3975_4    defaults
yaml                      0.1.7                h1bed415_2    defaults
zeromq                    4.2.5                hf484d3e_0    defaults
zict                      0.1.3                    py37_0    defaults
zlib                      1.2.11               h7b6447c_3    defaults
zstd                      1.3.7                h0b5b093_0    defaults
```
</p></details>

Commands run **after** the upgrade to 4.6.2
<details open><summary><code>`conda info`</code></summary><p>

```
(test_env) root@fd9798e8557f:~# conda info

     active environment : test_env
    active env location : /root/anaconda3/envs/test_env
            shell level : 2
       user config file : /root/.condarc
 populated config files :
          conda version : 4.6.2
    conda-build version : 3.16.3
         python version : 3.7.1.final.0
       base environment : /root/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-ppc64le
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-ppc64le
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-ppc64le
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /root/anaconda3/pkgs
                          /root/.conda/pkgs
       envs directories : /root/anaconda3/envs
                          /root/.conda/envs
               platform : linux-ppc64le
             user-agent : conda/4.6.2 requests/2.21.0 CPython/3.7.1 Linux/4.14.0-115.2.2.el7a.ppc64le ubuntu/18.04.1 glibc/2.27
                UID:GID : 0:0
             netrc file : None
           offline mode : False

(test_env) root@fd9798e8557f:~#
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
(test_env) root@fd9798e8557f:~# conda config --show-sources

(test_env) root@fd9798e8557f:~#


```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
(test_env) root@fd9798e8557f:~# conda list --show-channel-urls -n base
# packages in environment at /root/anaconda3:
#
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0                    py37_0    defaults
alabaster                 0.7.12                   py37_0    defaults
anaconda                  2018.12                  py37_0    defaults
anaconda-client           1.7.2                    py37_0    defaults
anaconda-project          0.8.2                    py37_0    defaults
asn1crypto                0.24.0                   py37_0    defaults
astroid                   2.1.0                    py37_0    defaults
astropy                   3.0.5            py37h7b6447c_0    defaults
atomicwrites              1.2.1                    py37_0    defaults
attrs                     18.2.0           py37h28b3542_0    defaults
babel                     2.6.0                    py37_0    defaults
backcall                  0.1.0                    py37_0    defaults
backports                 1.0                      py37_1    defaults
backports.os              0.1.1                    py37_0    defaults
backports.shutil_get_terminal_size 1.0.0                    py37_2    defaults
beautifulsoup4            4.6.3                    py37_0    defaults
bitarray                  0.8.3            py37h14c3975_0    defaults
bkcharts                  0.2                      py37_0    defaults
blas                      1.0                    openblas    defaults
blosc                     1.14.4               hdbcaa40_0    defaults
bokeh                     1.0.2                    py37_0    defaults
boto                      2.49.0                   py37_0    defaults
bottleneck                1.2.1            py37h035aef0_1    defaults
bzip2                     1.0.6                h14c3975_5    defaults
ca-certificates           2018.03.07           hf82bc7d_0    defaults
cairo                     1.14.12              h8948797_3    defaults
certifi                   2018.11.29               py37_0    defaults
cffi                      1.11.5           py37h9745a5d_0    defaults
chardet                   3.0.4                    py37_1    defaults
click                     7.0                      py37_0    defaults
cloudpickle               0.6.1                    py37_0    defaults
clyent                    1.2.2                    py37_1    defaults
colorama                  0.4.1                    py37_0    defaults
conda                     4.6.2                    py37_0    defaults
conda-build               3.16.3                   py37_0    defaults
conda-env                 2.6.0                         1    defaults
conda-verify              3.1.1                    py37_0    defaults
contextlib2               0.5.5                    py37_0    defaults
cryptography              2.4.2            py37h1ba5d50_0    defaults
curl                      7.63.0            hbc83047_1000    defaults
cycler                    0.10.0                   py37_0    defaults
cython                    0.29.2           py37he6710b0_0    defaults
cytoolz                   0.9.0.1          py37h14c3975_1    defaults
dask                      1.0.0                    py37_0    defaults
dask-core                 1.0.0                    py37_0    defaults
datashape                 0.5.4                    py37_1    defaults
decorator                 4.3.0                    py37_0    defaults
distributed               1.25.1                   py37_0    defaults
docutils                  0.14                     py37_0    defaults
entrypoints               0.2.3                    py37_2    defaults
et_xmlfile                1.0.1                    py37_0    defaults
expat                     2.2.6                he6710b0_0    defaults
fastcache                 1.0.2            py37h14c3975_2    defaults
filelock                  3.0.10                   py37_0    defaults
flask                     1.0.2                    py37_1    defaults
flask-cors                3.0.7                    py37_0    defaults
fontconfig                2.13.0               h9420a91_0    defaults
freetype                  2.9.1                h8a8886c_0    defaults
future                    0.17.1                   py37_0    defaults
get_terminal_size         1.0.0                         0    defaults
gevent                    1.3.7            py37h7b6447c_1    defaults
glib                      2.56.2               hd408876_0    defaults
glob2                     0.6                      py37_1    defaults
gmp                       6.1.2                h7f7056e_2    defaults
gmpy2                     2.0.8            py37h10f8cd9_2    defaults
greenlet                  0.4.15           py37h7b6447c_0    defaults
h5py                      2.8.0            py37h8d01980_0    defaults
hdf5                      1.10.2               hba1933b_1    defaults
heapdict                  1.0.0                    py37_2    defaults
html5lib                  1.0.1                    py37_0    defaults
icu                       58.2                 h64fc554_1    defaults
idna                      2.8                      py37_0    defaults
imageio                   2.4.1                    py37_0    defaults
imagesize                 1.1.0                    py37_0    defaults
importlib_metadata        0.6                      py37_0    defaults
ipykernel                 5.1.0            py37h39e3cac_0    defaults
ipython                   7.2.0            py37h39e3cac_0    defaults
ipython_genutils          0.2.0                    py37_0    defaults
ipywidgets                7.4.2                    py37_0    defaults
isort                     4.3.4                    py37_0    defaults
itsdangerous              1.1.0                    py37_0    defaults
jbig                      2.1                  h14c3975_0    defaults
jdcal                     1.4                      py37_0    defaults
jedi                      0.13.2                   py37_0    defaults
jinja2                    2.10                     py37_0    defaults
jpeg                      9b                   hcb7ba68_2    defaults
jsonschema                2.6.0                    py37_0    defaults
jupyter                   1.0.0                    py37_7    defaults
jupyter_client            5.2.4                    py37_0    defaults
jupyter_console           6.0.0                    py37_0    defaults
jupyter_core              4.4.0                    py37_0    defaults
jupyterlab                0.35.3                   py37_0    defaults
jupyterlab_server         0.2.0                    py37_0    defaults
kiwisolver                1.0.1            py37hf484d3e_0    defaults
krb5                      1.16.1               h597af5e_7    defaults
lazy-object-proxy         1.3.1            py37h14c3975_2    defaults
libarchive                3.3.3                h5d8350f_5    defaults
libcurl                   7.63.0            h20c2e04_1000    defaults
libedit                   3.1.20170329         h6b74fdf_2    defaults
libffi                    3.2.1                hf62a594_5    defaults
libgcc-ng                 8.2.0                h822a55f_1    defaults
libgfortran-ng            7.3.0                h822a55f_1    defaults
libopenblas               0.2.20               h9ac9557_7    defaults
libpng                    1.6.35               hbc83047_0    defaults
libsodium                 1.0.16               h1bed415_0    defaults
libssh2                   1.8.0                h1ba5d50_4    defaults
libstdcxx-ng              8.2.0                h822a55f_1    defaults
libtiff                   4.0.9                he85c1e1_1    defaults
libuuid                   1.0.3                h1bed415_2    defaults
libxcb                    1.13                 h1bed415_0    defaults
libxml2                   2.9.8                h26e45fe_1    defaults
libxslt                   1.1.32               h1312cb7_0    defaults
locket                    0.2.0                    py37_1    defaults
lxml                      4.2.5            py37hefd8a0e_0    defaults
lz4-c                     1.8.1.2              h14c3975_0    defaults
lzo                       2.10                 h0dabc4d_2    defaults
markupsafe                1.1.0            py37h7b6447c_0    defaults
matplotlib                3.0.2            py37h5429711_0    defaults
mccabe                    0.6.1                    py37_1    defaults
mistune                   0.8.4            py37h7b6447c_0    defaults
more-itertools            4.3.0                    py37_0    defaults
mpc                       1.1.0                h10f8cd9_1    defaults
mpfr                      4.0.1                hdf1c602_3    defaults
mpmath                    1.1.0                    py37_0    defaults
msgpack-python            0.5.6            py37h6bb024c_1    defaults
multipledispatch          0.6.0                    py37_0    defaults
nbconvert                 5.3.1                    py37_0    defaults
nbformat                  4.4.0                    py37_0    defaults
ncurses                   6.1                  he6710b0_1    defaults
networkx                  2.2                      py37_1    defaults
nltk                      3.4                      py37_1    defaults
nose                      1.3.7                    py37_2    defaults
notebook                  5.7.4                    py37_0    defaults
numexpr                   2.6.7            py37h98f89e9_0    defaults
numpy                     1.14.5           py37h2aefc1b_4    defaults
numpy-base                1.14.5           py37h2b20989_4    defaults
numpydoc                  0.8.0                    py37_0    defaults
odo                       0.5.1                    py37_0    defaults
olefile                   0.46                     py37_0    defaults
openblas                  0.2.20                        7    defaults
openblas-devel            0.2.20                        7    defaults
openpyxl                  2.5.12                   py37_0    defaults
openssl                   1.1.1a               h7b6447c_0    defaults
packaging                 18.0                     py37_0    defaults
pandas                    0.23.4           py37h04863e7_0    defaults
pandoc                    2.0.0.1                       1    defaults
pandocfilters             1.4.2                    py37_1    defaults
parso                     0.3.1                    py37_0    defaults
partd                     0.3.9                    py37_0    defaults
patchelf                  0.9                  he6710b0_3    defaults
path.py                   11.5.0                   py37_0    defaults
pathlib2                  2.3.3                    py37_0    defaults
patsy                     0.5.1                    py37_0    defaults
pcre                      8.42                 hf484d3e_0    defaults
pep8                      1.7.1                    py37_0    defaults
pexpect                   4.6.0                    py37_0    defaults
pickleshare               0.7.5                    py37_0    defaults
pillow                    5.3.0            py37h0d2faf8_0    defaults
pip                       18.1                     py37_0    defaults
pixman                    0.34.0               h1f8d8dc_3    defaults
pkginfo                   1.4.2                    py37_1    defaults
pluggy                    0.8.0                    py37_0    defaults
ply                       3.11                     py37_0    defaults
prometheus_client         0.5.0                    py37_0    defaults
prompt_toolkit            2.0.7                    py37_0    defaults
psutil                    5.4.8            py37h7b6447c_0    defaults
ptyprocess                0.6.0                    py37_0    defaults
py                        1.7.0                    py37_0    defaults
pycodestyle               2.4.0                    py37_0    defaults
pycosat                   0.6.3            py37h14c3975_0    defaults
pycparser                 2.19                     py37_0    defaults
pycrypto                  2.6.1            py37h14c3975_9    defaults
pycurl                    7.43.0.2         py37h1ba5d50_0    defaults
pyflakes                  2.0.0                    py37_0    defaults
pygments                  2.3.1                    py37_0    defaults
pylint                    2.2.2                    py37_0    defaults
pyodbc                    4.0.25           py37he6710b0_0    defaults
pyopenssl                 18.0.0                   py37_0    defaults
pyparsing                 2.3.0                    py37_0    defaults
pysocks                   1.6.8                    py37_0    defaults
pytables                  3.4.4            py37ha205bf6_0    defaults
pytest                    4.0.2                    py37_0    defaults
pytest-arraydiff          0.3              py37h39e3cac_0    defaults
pytest-astropy            0.5.0                    py37_0    defaults
pytest-doctestplus        0.2.0                    py37_0    defaults
pytest-openfiles          0.3.1                    py37_0    defaults
pytest-remotedata         0.3.1                    py37_0    defaults
python                    3.7.1                h4134adf_7    defaults
python-dateutil           2.7.5                    py37_0    defaults
python-libarchive-c       2.8                      py37_6    defaults
pytz                      2018.7                   py37_0    defaults
pywavelets                1.0.1            py37hdd07704_0    defaults
pyyaml                    3.13             py37h14c3975_0    defaults
pyzmq                     17.1.2           py37h14c3975_0    defaults
readline                  7.0                  h7b6447c_5    defaults
requests                  2.21.0                   py37_0    defaults
ruamel_yaml               0.15.46          py37h14c3975_0    defaults
scikit-image              0.14.1           py37he6710b0_0    defaults
scikit-learn              0.19.1           py37h6cfcb94_0    defaults
scipy                     1.1.0            py37h9c1e066_0    defaults
seaborn                   0.9.0                    py37_0    defaults
send2trash                1.5.0                    py37_0    defaults
setuptools                40.6.3                   py37_0    defaults
simplegeneric             0.8.1                    py37_2    defaults
singledispatch            3.4.0.3                  py37_0    defaults
six                       1.12.0                   py37_0    defaults
snappy                    1.1.7                h1532aa0_3    defaults
snowballstemmer           1.2.1                    py37_0    defaults
sortedcollections         1.0.1                    py37_0    defaults
sortedcontainers          2.1.0                    py37_0    defaults
sphinx                    1.8.2                    py37_0    defaults
sphinxcontrib             1.0                      py37_1    defaults
sphinxcontrib-websupport  1.1.0                    py37_1    defaults
sqlalchemy                1.2.15           py37h7b6447c_0    defaults
sqlite                    3.26.0               h7b6447c_0    defaults
statsmodels               0.9.0            py37h035aef0_0    defaults
sympy                     1.3                      py37_0    defaults
tblib                     1.3.2                    py37_0    defaults
terminado                 0.8.1                    py37_1    defaults
testpath                  0.4.2                    py37_0    defaults
tk                        8.6.8                hbc83047_0    defaults
toolz                     0.9.0                    py37_0    defaults
tornado                   5.1.1            py37h7b6447c_0    defaults
tqdm                      4.28.1           py37h28b3542_0    defaults
traitlets                 4.3.2                    py37_0    defaults
unicodecsv                0.14.1                   py37_0    defaults
unixodbc                  2.3.7                h2c717c6_0    defaults
urllib3                   1.24.1                   py37_0    defaults
wcwidth                   0.1.7                    py37_0    defaults
webencodings              0.5.1                    py37_1    defaults
werkzeug                  0.14.1                   py37_0    defaults
wheel                     0.32.3                   py37_0    defaults
widgetsnbextension        3.4.2                    py37_0    defaults
wrapt                     1.10.11          py37h14c3975_2    defaults
xlrd                      1.2.0                    py37_0    defaults
xlsxwriter                1.1.2                    py37_0    defaults
xlwt                      1.3.0                    py37_0    defaults
xz                        5.2.4                h14c3975_4    defaults
yaml                      0.1.7                h1bed415_2    defaults
zeromq                    4.2.5                hf484d3e_0    defaults
zict                      0.1.3                    py37_0    defaults
zlib                      1.2.11               h7b6447c_3    defaults
zstd                      1.3.7                h0b5b093_0    defaults
(test_env) root@fd9798e8557f:~#

```
</p></details>

**Comment by jjhelmus** [comment]
I believe this is related to #8301, something is problematic with the blas package for the linux-ppc64le platform with conda 4.6.

**Comment by jjhelmus** [comment]
Running the problematic command with debugging turned on gives some hints on what is happening:

```
#CONDA_DEBUG=1 conda install 'numpy>=1.12.0'
...
DEBUG conda.core.solve:_run_sat(469): conflicting specs: 
  - blas
DEBUG conda.resolve:get_reduced_index(354): Retrieving packages for: 
  - blas
  - ca-certificates
  - libedit
  - libffi
  - libgcc-ng
  - libgfortran-ng
  - libopenblas
  - libstdcxx-ng
  - ncurses
  - numpy-base
  - numpy[version='>=1.12.0']
  - openssl
  - python=3.6
  - readline
  - sqlite
  - tk
  - xz
  - zlib
```

The numpy-1.12.0-py36_0 package from the free sub-channel does not depend on blas and therefore is selected.

**Comment by jjhelmus** [comment]
The issue here is that the `blas` package in `pkgs/main` and `pkgs/free` and considered by the solver to be the same package (see #8302) while at the same time the solver requires only of them to be installed in an environment.  This creates an Unsatisfiable SAT condition.  

Removing `pkgs/free` from the list of considered channel is a workaround until this can be properly fixes.  This can be accomplished by running:

```
conda config --add default_channels https://repo.anaconda.com/pkgs/main
conda config --add default_channels https://repo.anaconda.com/pkgs/r
```

**Comment by wdirons** [comment]
Thank you @jjhelmus for getting to the root cause of this issue.

The workaround works for me. I'll close this issue as it will be fixed under #8302

**Comment by jjhelmus** [comment]
An update to the blas package was made so that the `blas` packages in the `pkgs/main` and `pkgs/free` no longer conflict with each other.  The workaround mentioned in the early comment is no longer required.
